packages/firestore/test/integration/util/helpers.ts

```diff
@@ -182,7 +184,40 @@ export async function withTestDbsSettings(
   const dbs: Firestore[] = [];
 
   for (let i = 0; i < numDbs; i++) {
-    const db = newTestFirestore(projectId, /* name =*/ undefined, settings);
+    const db = newTestFirestore(newTestApp(projectId), settings);
+    if (persistence) {
+      await enableIndexedDbPersistence(db);
+    }
+    dbs.push(db);
+  }
+
+  try {
+    await fn(dbs);
+  } finally {
+    for (const db of dbs) {
+      await terminate(db);
+      if (persistence) {
+        await clearIndexedDbPersistence(db);
+      }
+    }
+  }
+}
+
+export async function withNamedTestDbs(
+  persistence: boolean,
+  dbNames: string[],
+  fn: (db: Firestore[]) => Promise<void>
+): Promise<void> {
+  // Named DBs requires preparing project with DBs beforehand.
```

Comment:
This comment does not read well..please fix.

Reply:
Fixed comment